lib/std/winapis.nim

```diff
@@ -0,0 +1,14 @@
+## An inplacement for `std/winlean` with presice and stable types definitions.
```

Comment:
what do you mean by inplacement? replacement? (couldn't find a definition onlline)